BUILDING.md

```diff
@@ -0,0 +1,68 @@
+## Build the development environment
+
+The first prerequisite of properly building SwarmKit targets is to have a Go
```

Comment:
Suggested edit: 
To build Swarmkit, you must set up a Go development environment. For full instructions see [How to Write Go Code](https://golang.org/doc/code.html).

Briefly, to install a Go environment on Linux:
1. Download Go from [https://golang.org/dl/](https://golang.org/dl/).
2. Expand the package and move it to `/usr/local`:
```sh
$ tar xzvf go1.5.4.linux-amd64.tar.gz
$ sudo mv go /usr/local
```
> **Note**: On Ubuntu, do not use `apt-get` to install Go because its repositories tend
> to include older versions of Go. Instead, install the latest Go manually using the
> instructions provided on the Go site.

3. Create a Go project directory in your home directory:
```$ sh
$ mkdir ~/gocode    # any name is fine
```
4. Add the following variables to your `.bashrc`:
```sh
$ export GO15VENDOREXPERIMENT=1 # only required for go 1.5.x
$ export GOROOT=/usr/local/go
$ export GOPATH=~/gocode
$ export PATH=$PATH:$GOPATH/bin:$GOROOT/bin
```
5. Close and reopen your terminal.